### PR TITLE
Fix integration tests

### PR DESCRIPTION
**Pull request by oplantalech**
Fix integration tests that were failing due to the recent update of the seed database:

- Gene ACPP is now ACP3
- microRNA gene MIR4714 is not in the seed database.
- Use public database for the OncoKB import integration test

### CODE REVIEW

File: test/integration/integration_test_oncokb_import.sh [hunk 1]
```diff
@@ -6,7 +6,7 @@
 cd "$PORTAL_SOURCE_DIR"/core/src/main/scripts/importer
 python3 ./metaImport.py \
   --study_directory="$PORTAL_SOURCE_DIR"/core/src/test/scripts/test_data/study_oncokb_import \
-  --portal_info_dir="$PORTAL_SOURCE_DIR"/core/src/test/scripts/test_data/api_json_system_tests \
+  --url_server="https://www.cbioportal.org" \
   --import_oncokb \
```
Comment (inodb) [inline test/integration/integration_test_oncokb_import.sh:10]: @oplantalech should we update the test_data as well?

**Review by inodb — APPROVED** [review]
@oplantalech Thanks so much for fixing this!

File: test/integration/integration_test_oncokb_import.sh (hunk 1 above)
Comment (inodb) [reply test/integration/integration_test_oncokb_import.sh:10]: Like this one `"$PORTAL_SOURCE_DIR"/core/src/test/scripts/test_data/api_json_system_tests`

Comment (inodb) [reply test/integration/integration_test_oncokb_import.sh:10]: and if not - do we just delete that folder?

Comment (oplantalech) [reply test/integration/integration_test_oncokb_import.sh:10]: @inodb we need to address this with the datahub team. We use some genes `A1CF` (Entrez ID: `29974`), `A2M` (Entrez ID: `2`) and `A2ML1` (Entrez ID: `144568`) that are not in the new seed but they are in the public database. I don't know if it is a mistake or not, we need to discuss it.

Ideally then we should update indeed these test_data

**Pull request merged**